enumerables.rb

```diff
@@ -0,0 +1,155 @@
+module Enumerable
+    def my_each
+        for i in 0...self.length do
+            yield(self[i])
+        end
+    end
+    
```

Comment:
- [ ]  my_each is not handling ranges and hashes correctly when a block is given.